apps/judicial-system/api/src/app/modules/case/case.resolver.ts

```diff
@@ -261,6 +262,8 @@ export class CaseResolver {
   ): Promise<Notification[]> {
     const { id } = theCase
 
+    this.logger.debug(`Resolving notifications for case ${id}`)
```

Comment:
Did you mean to leave this in here?

Reply:
Yes, we usually debug log entry points to resolvers and controllers. It was missing here.